src/frame/worker/crypto/recrypt/RecryptWasm.ts

```diff
@@ -248,6 +248,13 @@ export const createRequestSignature = (
     };
 };
 
+/**
+ * Generate a schnorr signature using the group's private key over the group's provided id
+ */
+export const schnorrSignUtf8String = (privateKey: Uint8Array, publicKey: PublicKey<string>, string: string) => {
```

Comment:
Sorry I didn't notice this before, but we should make this take a `PublicKey<Uint8Array>` instead of as a string. It's inconsistent that we take the private key as bytes but the public key as a string. Callers should just convert it from a string to bytes before calling into here.

Reply:
done